nidmviewer/convert.py

```diff
@@ -24,7 +25,10 @@ def parse_coordinates(coordinates):
     count=0
     coordinate_df = pandas.DataFrame(columns=["x","y","z"])
     for coordinate in coordinates:
-        coordinate_df.loc[count] = [x.strip() for x in coordinate.strip("]").strip("[").split(",")]
+        if not isinstance(coordinate, float64) or not isnan(coordinate):
```

Comment:
I'm confused why you added this check for float / nan - the coordinates come from a data frame column (coordinate) in the peaks table that is always a list of strings:

      ['[ -8.72, -96.4, 8.03 ]',
       '[ 4.83, -92.6, -5.96 ]',
       ....
       '[ 20.9, 31.5, -17.5 ]']

Did something change that I'm not aware of?